.github/workflows/release.yaml

```diff
@@ -346,8 +358,12 @@ jobs:
           profile: minimal
           toolchain: ${{ env.RUST_STABLE }}
           target: wasm32-unknown-unknown
+          components: ${{ matrix.components }}
```

Comment:
*Not sure*, but wouldn't it be better for caching to just  always fetch `stable` + `rust-src`?

Reply:
i'm not sure either, i'll change it to always fetch the components, we can change it later on anyways.